File: BankApp/Bank Account/Account Transfers and Transactions/Steps - Ruby/StepDefinition-AccountT&Ts-SearchT&Ts.rb
Given /^the following user:$/ do 
    # do something
end

Given /^I am logged in as Bob$/ do 
    # do something
end

When /^I search for (.+) with (.+)$/ do |criteria, searchvalue|
    # do something
end

Then /^I am informed: \"([^\"]*)\\".$/ do |therearenoresultsthatmatchyourcriteria|
    # do something
end

Then /^I should be able to see ID of the transaction, the (.+), the (.+) and (.+)$/ do |type, value, balance|
    # do something
end

And /^the following transactions:$/ do 
    # do something
end

And /^my balance is £1000.00$/ do 
    # do something
end

File: BankApp/Bank Account/Account Transfers and Transactions/Steps - JavaScript/StepDefinition-AccountT&Ts-SearchT&Ts.js
module.exports = function () {

  this.Given(/^the following user:$/, function (callback) {
    callback.pending();
  });

  this.Given(/^I am logged in as Bob$/, function (callback) {
    callback.pending();
  });

  this.When(/^I search for (.+) with (.+)$/, function (criteria, searchvalue, callback) {
    callback.pending();
  });

  this.Then(/^I am informed: \"([^\"]*)\".$/, function (therearenoresultsthatmatchyourcriteria, callback) {
    callback.pending();
  });

  this.Then(/^I should be able to see ID of the transaction, the (.+), the (.+) and (.+)$/, function (type, value, balance, callback) {
    callback.pending();
  });

  this.And(/^the following transactions:$/, function (callback) {
    callback.pending();
  });

  this.And(/^my balance is £1000.00$/, function (callback) {
    callback.pending();
  });

};
